### PR TITLE
Fixed #5732, #5735: fixed `box-shadow` and `background-color` in quick-open menu 

### LINKED ISSUE #5732: [quick-open] grey line when scrolling down quick-open menu in dark theme
In the **dark theme**, when scrolling down the quick-open menu, I notice there is a thick grey line appearing at the top position, which looks strange.

It is probably the same grey shadow line used in the **light theme**. But it should not be part of the quick-open menu UI in the dark theme.

theia-quickopen-line

image

### PR DESCRIPTION
**Pull request by fangnx**
- This patch added dark-theme support for the `box-shadow` on the quick-open menu. To be aligned with VS Code and the current UI, the top divider (shadow) would be black in the dark theme and light-grey in the light theme.
- Also, aligned `background-color` of the quick-open menu with other widgets (e.g. the side-panel widget). This would also make the top shadow line more visible in the dark mode.

### Before
theia-quickopen-line

### After
image

### CODE REVIEW

File: packages/core/src/browser/style/variables-dark.useable.css [hunk 1]
```diff
@@ -196,6 +196,7 @@ is not optimized for dense, information rich UIs.
   --theia-scrollbar-rail-color: transparent;
   --theia-scrollbar-active-thumb-color: hsla(0,0%,61%,.4);
   --theia-scrollbar-active-rail-color: transparent;
+  --theia-scrollbar-box-shadow-color: black;
```
Comment (vince-fugnitto) [inline packages/core/src/browser/style/variables-dark.useable.css]: For the sake of consistency, can you provide the `hex` value instead like other colors.

Comment (vince-fugnitto) [reply packages/core/src/browser/style/variables-dark.useable.css]: Also, we should probably align the **background-color** of the `quick-open` to match that of vscode.
Currently, on dark themes the shadow is unnoticeable. 

The **background-color** should be the same as docked widgets in the sidebar, so the value should be
```css
--theia-layout-color1
```

Comment (vince-fugnitto) [reply packages/core/src/browser/style/variables-dark.useable.css]: I created an issue for it https://github.com/theia-ide/theia/issues/5735, but I believe it should be addressed with this PR, you can add a commit on top of your current one to include it :)

Comment (fangnx) [reply packages/core/src/browser/style/variables-dark.useable.css]: Sure, I will do it ;)

**Comment by vince-fugnitto** [comment]
@jbicker would you like to take a look or shall I merge?

**Comment by jbicker** [comment]
merge :)

**Pull request merged**